setup.py

```diff
@@ -68,22 +69,23 @@ def parse_requirements_file(path):
     maintainer_email=metadata.email,
     license=metadata.license,
     url=metadata.url,
+    python_requires=">=3.8.0",
```

Comment:
the project can't assume it's forward compatible though; it's already known that it definitely won't be compatible with the python ver which releases in just over 4 years from now and that's only accounting for one PEP